tests/dummy/config/environment.js

```diff
@@ -40,8 +40,6 @@ module.exports = function(environment) {
     ENV.baseURL = '/';
     ENV.locationType = 'none';
 
-    ENV.EmberENV.RAISE_ON_DEPRECATION = true;
```

Comment:
I wouldn't remove `RAISE_ON_DEPRECATION` and instead fix the `Ember.keys` deprecation:
`forEach(Object.keys(component) || keys(component), (key) => {`

not tested but should work.